src/cascadia/TerminalApp/Resources/en-US/Resources.resw

```diff
@@ -334,4 +334,16 @@
     <value>Command Prompt</value>
     <comment>This is the name of "Command Prompt", as localized in Windows. The localization here should match the one in the Windows product for "Command Prompt"</comment>
   </data>
+  <data name="CloseTabDialog.PrimaryButtonText" xml:space="preserve">
+    <value>Close Tab</value>
+    <comment>This is the name of "Command Prompt", as localized in Windows. The localization here should match the one in the Windows product for "Command Prompt"</comment>
+  </data>
+  <data name="CloseTabDialog.CloseButtonText" xml:space="preserve">
+    <value>Cancel</value>
+    <comment>This is the name of "Command Prompt", as localized in Windows. The localization here should match the one in the Windows product for "Command Prompt"</comment>
+  </data>
+  <data name="CloseTabDialog.Title" xml:space="preserve">
+    <value>Are you sure you want to close this tab?</value>
+    <comment>This is the name of "Command Prompt", as localized in Windows. The localization here should match the one in the Windows product for "Command Prompt"</comment>
+  </data>

```

Comment:
The comment on all of these is wrong.

Also, please make these changes in Visual Studio. Just use Solution Explorer to get to this resw file (should be under the TerminalAppLib project). Then you should get a nice table to be able to do this in. Make and save the changes there.

Reply:
Oops, just copy and pasted over. I forgot to change comments. Sorry!

Reply:
It's now fixed! Thx.